test/public/runs/runsPerSimulationPass.overview.test.js

```diff
@@ -177,26 +168,23 @@ module.exports = () => {
     });
 
     it('notifies if table loading returned an error', async () => {
-        await goToPage(page, 'runs-per-simulation-pass', { queryParameters: { simulationPassId: 2 } });
-
         // eslint-disable-next-line no-return-assign, no-undef
         await page.evaluate(() => model.runs.perSimulationPassOverviewModel.pagination.itemsPerPage = 200);
 
         // We expect there to be a fitting error message
         const expectedMessage = 'Invalid Attribute: "query.page.limit" must be less than or equal to 100';
         await expectInnerText(page, '.alert-danger', expectedMessage);
+        // eslint-disable-next-line no-return-assign, no-undef
+        await page.evaluate(() => model.runs.perSimulationPassOverviewModel.pagination.itemsPerPage = 10);
```

Comment:
Doing so you might want to wait for the table to have a length of 10?

Reply:
Still, you need to wait for the update to avoid flaky test. You need to wait for the table to have the expected size anyhow

Reply:
ok, applied